cis/settings.py

```diff
@@ -4,3 +4,4 @@
 ARN_MASTER_KEY = config('CIS_ARN_MASTER_KEY')
 DYNAMODB_TABLE = config('CIS_DYNAMODB_TABLE')
 KINESIS_STREAM_ARN = config('CIS_KINESIS_STREAM_ARN')
+LAMBDA_VALIDATOR_ARN = config('CIS_LAMBDA_VALIDATOR_ARN')
```

Comment:
Can we make this an optional config parameter since it's not always needed.  In other words always fall back to None?